library/OrderCloud.Catalyst/Integrations/Exceptions/IntegrationAuthFailedException.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace OrderCloud.Catalyst
+{
+	public class IntegrationAuthFailedException : CatalystBaseException
+	{
+		public IntegrationAuthFailedException(OCIntegrationConfig config, string requestUrl) : base(
+			"IntegrationAuthorizationFailed",
+			$"Authentication to 3rd party service \"{config.ServiceName}\" failed. Check your config credentials.",
+			new IntegrationAuthFailedError() 
+			{
+				ServiceName = config.ServiceName,
+				RequestUrl = requestUrl,
+			}, 
+			400) {}
```

Comment:
@oliverheywood451, should we use the `HttpStatusCode` enum of the `System.Net` instead of integer? I see that the base class accepts the `int` and maybe in the future can improve this area, what do you think?